### PR TITLE
Laravel pipelines tracer supporting configurable handler method.

### PR DESCRIPTION
**Pull request by labbati**
### Description

Refer to #157 for an explanation of the issue.

### Readiness checklist
- [x] [docs/changelog.md](Changelog entry) added, if necessary
- [x] Tests added for this feature/bug

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ x-aliases:
       tty: true
       volumes:
         - .:/home/circleci/app
+        - .composer:/home/circleci/.composer
```
Comment (labbati) [inline docker-compose.yml]: This way switching between different versions and tested frameworks is much more fast.

Comment (SammyK) [reply docker-compose.yml]: Nice! <3

File: tests/Integration/Frameworks/Laravel/Version_5_7/tests/Integration/PipeTracingTest.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+use DDTrace\Tests\Integration\Common\TracerTestTrait;
+use DDTrace\Tests\Integration\Frameworks\Util\CommonScenariosDataProviderTrait;
+use DDTrace\Tests\Integration\Frameworks\Util\Request\GetSpec;
+use DDTrace\Tests\Integration\Frameworks\Util\Request\RequestSpec;
+use Illuminate\Pipeline\Pipeline;
+use Tests\TestCase;
+
+/**
+ */
+class PipelineTracingTest extends TestCase
```
Comment (SammyK) [inline tests/Integration/Frameworks/Laravel/Version_5_7/tests/Integration/PipeTracingTest.php]: The file name `PipeTracingTest.php` doesn't match the class name. :)

Comment (labbati) [reply tests/Integration/Frameworks/Laravel/Version_5_7/tests/Integration/PipeTracingTest.php]: Good catch! Fixed it

**Comment by labbati** [comment]
@SammyK I fixed the class name, whenever you are available I would like to have your final approval

**Pull request merged**